src/full/Agda/TypeChecking/Primitive/Cubical.hs

```diff
@@ -526,7 +526,7 @@ primTransHComp cmd ts nelims = do
               -- compData knows what to do for the general cases.
               Datatype{dataPars = pars, dataIxs = ixs, dataPathCons = pcons, dataTransp = mtrD}
                 | and [null pcons && ixs == 0 | DoHComp  <- [cmd]], Just as <- allApplyElims es ->
-                  compData mtrD ((not $ null $ pcons) || ixs > 0) (pars+ixs) cmd l (as <$ t) sbA sphi u u0
+                  compData mtrD (not (null $ pcons) || ixs > 0) (pars+ixs) cmd l (as <$ t) sbA sphi u u0
```

Comment:
Shouldn't the other `$` go as well? `(null $ pcons)`

Reply:
Yeah I missed that in my review before committing. Fixed now in c4985b1dfaed834ebfa50af6922e57dc45fac891.